backend/src/repo/clientAppRepository.js

```
@@ -195,42 +197,44 @@ const getSessions = async (schema = DEFAULT_SCHEMA, userId) => {
-   const pool = await check(schema);
+   const pool = await getPool(schema);
    return sessionRepo.deleteSessionBySessionId(pool, sessionId);
 };
 
-export default {
-   // Users
+export const userRepo = {
    createUser,
    getUsers,
    getUser,
    getUserByEmail,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'userRepo'. Consider to delete it or rename it.

'userRepo' is defined here:


(lint/suspicious/noRedeclare)